app/assets/stylesheets/components/rad-notification/_settings.scss

```diff
@@ -0,0 +1,5 @@
+$rad-notification-radius: 5px;
+$color-rad-notification-warning: #F89E54;
```

Comment:
We've moved this to the main _settings file in RAD. Is it ok to leave this here until there is a need for it elsewhere, or would you prefer it to be added to Dough just now?